conda.recipe/meta.yaml

```diff
@@ -1,11 +1,11 @@
 # Conda Recipe for Tethys Platform
 # WARNING: THIS IS A GENERATED FILE. DO NOT EDIT.
 # TO CHANGE THIS FILE, SEE $TETHYS_SRC/tethys_cli/gen_templates/meta yaml
-{% set data = load_setup_py_data() %}
+
 
 package:
   name: tethys-platform
-  version: {{ data['version'] }}
+  version: 3.4.1.dev24+gb092a0af.d20220421
```

Comment:
Should this version be hard coded?

Reply:
This was previously committed so we'll leave it for now.